docs/multicluster/user-guide.md

```diff
@@ -559,6 +559,42 @@ In future releases, some additional tooling may become available to automate the
 creation of ResourceExports for ACNPs, and provide a user-friendly way to define
 Multi-cluster NetworkPolicies to be enforced in the ClusterSet.
 
+## Antrea Multicluster Pod to Pod Connectivity
+Since Antrea v1.9.0, Multi-cluster allows traffic between Pods in different clusters. If you want to enable this feature,
+the Pod CIDRs must be set in the ConfigMap `antrea-mc-controller-config` like below. You can edit the config in
+[antrea-multicluster-member.yml](../../multicluster/build/yamls/antrea-multicluster-member.yml) or using `kubectl edit`.
```

Comment:
You can edit [antrea-multicluster-member.yml] (../../multicluster/build/yamls/antrea-multicluster-member.yml) or use `kubectl edit` to change the ConfigMap.

Reply:
Can we add an example command of using `kubectl edit`? Then start a new paragraph for this sentence after the YAML example.

Reply:
Add a new example command using kubectl edit.